src/Game/zTaxi.h

```diff
@@ -1,6 +1,67 @@
 #ifndef ZTAXI_H
 #define ZTAXI_H
 
+#include <types.h>
+#include "../Core/x/xBase.h"
+#include "../Core/x/xDynAsset.h"
+#include "zEnt.h"
 
+struct callback
+{
+};
+
+struct taxi_asset : xDynAsset
+{
+    uint32 marker;
+    uint32 cameraID;
+    uint32 portalID;
+    uint32 talkBoxID;
+    uint32 textID;
+    uint32 taxiID;
+    float32 invDelay;
+    float32 portalDelay;
+};
+
+struct zTaxi : xBase
+{
+    taxi_asset* basset;
+    zEnt* taxi;
+    xVec3 pos;
+    uint32 currState;
+    uint32 prevState;
+    float32 portalTimer;
+    float32 invTimer;
+    float32 radius;
+};
+
+struct ztalkbox : xBase
+{
+    enum answer_enum
+    {
+        ANSWER_CONTINUE,
+        ANSWER_YES,
+        ANSWER_NO,
+        ANSWER_NONE
+    };
+};
+
+struct taxiCB : callback
+{
+    //zTaxi* taxi;
+    //answer_enum answer;
```

Comment:
Similar to my other comment, please import structs exactly how they appear in the dwarf data (ONLY unless there is a reason to delete a struct member, i.e. the dwarf data is incorrect between PS2 and gamecube).

Reply:
That, I commented out because I couldn't understand why it didn't compile, only to see I forgot ztalkbox:: in front of the answer enum.

The answer enum is using 1 and not both, as defining both leads to errors in compilation and makes no sense as they are nearly the same.